src/index.html

```diff
@@ -11,7 +11,173 @@
+          <input
+            type="checkbox"
+            id="agreement"
+            name="agreement"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form">
+        <legend class="legend">Registration</legend>
+        <div class="inputThing">
+          <label for="email">E-mail: </label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </div>
+        <div class="inputThing">
+          <label for="password">Password: </label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            autocomplete="off"
+            minlength="8"
+            maxlength="36"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form">
+        <legend class="legend">An interesting fact about you!</legend>
+        <div class="inputThing">
+          Do you love cats?
+          <input
+            type="radio"
+            name="cats"
+            id="catsYes"
+            value="catsYes"
+            checked
+          >
+          <label for="catsYes">Yes</label>
+          <input
+            type="radio"
+            name="cats"
+            id="catsNo"
+            value="catsNo"
+          >
+          <label for="catsNo">No</label>
+        </div>
+        <div class="inputThing">
+          <label for="color"> What is your favorite color? </label>
+          <input
+            type="color"
+            name="color"
+            id="color"
+          >
+        </div>
+        <div class="inputThing">
+          <label for="time"> What time do you go to bed? </label>
+          <input
+            type="time"
+            id="time"
+            name="time"
+          >
+        </div>
+        <div class="inputThing">
+          <label for="cars">What are your favorite brand of cars? </label>
+          <select
+            id="cars"
+            name="cars"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </div>
+        <div class="inputThing">
+          <label for="work">How do you rate our work? </label>
+          <input
+            type="range"
+            id="work"
+            name="work"
+            min="1"
+            max="10"
+            value="5"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form">
+        <legend class="legend">Additional info:</legend>
+        <div class="inputThing">
+          <label for="comment">Comments: </label>
+          <textarea
+            id="comment"
+            name="comment"
+            rows="2"
+            autocomplete="off"
+          >
+          </textarea>
```

Comment:
use this trick for textarea to avoid empty space when click on input
```suggestion
          ></textarea>
```

Reply:
Thanks, I didn't know that. But in this case, the linter yells at me because "Tag’s closing bracket should be on the same level as opening one". I'm submitting the updated code with all suggestions implemented except this one (otherwise, it won't let me).